src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -160,7 +160,39 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.reloadPage();
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditSuccess.html");
 
-
+        ______TS("test two 'change' links' functionality to expand uncommon settings panels");
+        
+        // test uncommon settings for 'send emails'
+        feedbackEditPage.clickEditUncommonSettingsSendEmailsButton();
+        feedbackEditPage.verifyEditSessionBoxIsEnabled();
+        feedbackEditPage.toggleClosingSessionEmailReminderCheckbox();
+        feedbackEditPage.clickSaveSessionButton();
+        feedbackEditPage.reloadPage();
```

Comment:
What I meant was continuously do a GET until you hit a feedback session that has changed. But this is inconsistent, if anything, so I'm reluctant to go down this path as well.

> I believe we have a standard way to make a test wait for a specific condition.

@damithc The specific conditions are limited to what can be sensed by Selenium Webdriver, i.e. an element's appearance/disappearance/visibility etc. Anything in the Datastore is beyond the reach of Selenium, and rightly so; why would a Webdriver have anything to do with the database?